call_test.go

```diff
@@ -61,6 +61,9 @@ func TestCallFromContext(t *testing.T) {
 			ShardKey:        "one",
 			RoutingKey:      "two",
 			RoutingDelegate: "three",
+			Features: transport.RequestFeatures{
+				AcceptResponseError: true,
```

Comment:
As I’m reading this, it gets clearer that the "Rpc-Accept-Reaponse-Error" header might be better as "Rpc-Accept-Application-Error" since if affects how we interpret the "Rpc-Application-Error" response header. "Response-Error" leads the question "What kind of error isn’t a response error".

Reply:
Disregard this.

Content negotiation needs to have an `"Accepts-*"` request header and a `"*"` response header, like `"Accepts-Encoding: gzip, inflate"` request header and `"Encoding: gzip"` response header. The server should signal whether it has elected to take advantage of the feature. Existing servers will continue returning error messages, and new servers will see the feature and response "Oh by the way, the error object is in the body".

Since "Rpc-Application-Error" exists regardless of whether we’re using the response body error feature, we need a different dual.

We could have "Rpc-Accepts-Response-Error" and "Rpc-Response-Error", but I note that "What kind of error isn't a response".

Maybe "Rpc-Accepts-Error-Content" and "Rpc-Error-Content"?

Reply:
I decided on `Rpc-Accepts-Both-Response-Error` to be more specific - error content isn't really what's going on either, as there is error content regardless, I want to say that both a response and error are potentially returned.